### PR TITLE
Shibboleth: Load username from config for UI

### PR DESCRIPTION
**Pull request by guruz**
For https://github.com/owncloud/enterprise/issues/1034

### CODE REVIEW

File: src/gui/creds/shibbolethcredentials.cpp [hunk 1]
```diff
@@ -129,9 +134,6 @@ bool ShibbolethCredentials::ready() const
 
 void ShibbolethCredentials::fetchFromKeychain()
 {
-    if (_user.isEmpty()) {
-        _user = _account->credentialSetting(QLatin1String(userC)).toString();
```
Comment (ogoffart) [inline src/gui/creds/shibbolethcredentials.cpp]: The HTTP crendatials still does fetch the user from this point.  I think we should still be consistent with the HTTP credentials.  So perhaps you should keep it there.  Or remove it as well from the HTTP credentials.

**Comment by guruz** [comment]
@ogoffart commit modified..

**Comment by guruz** [comment]
@ogoffart I've tested logging in with another existing username, it properly asks me to use the $correct_username via a dialog

**Comment by ogoffart** [comment]
perfect!  :+1:

**Pull request merged**